scripts/cargo-install-all.sh

```diff
@@ -23,13 +23,17 @@ maybeRustVersion=
 installDir=
 buildVariant=release
 maybeReleaseFlag=--release
+testnetMode=
 
 while [[ -n $1 ]]; do
   if [[ ${1:0:1} = - ]]; then
     if [[ $1 = --debug ]]; then
       maybeReleaseFlag=
       buildVariant=debug
       shift
+    elif [[ $1 = --testnet ]]; then
```

Comment:
How about naming this something like `--validator-only`.  It's not really testnet specific, and would be generally useful for mainnet-beta validators that use this script to build their validator from source as well

Reply:
Sure thing, updated